Makefile

```diff
@@ -96,3 +104,15 @@ release/image:
 	APP_VERSION=$(APP_VERSION) \
 	bazel run --stamp --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
 		//:push_operator_image 
+
+#
+# Dev target that
+#
+.PHONY: dev/syncdeps
+dev/syncdeps:
+	bazel run //hack:update-deps \
+	bazel run //hack:update-bazel \
+	bazel run //:gazelle -- update-repos -from_file=go.mod
```

Comment:
What causes use to need the `update-repos`?  Also, should we have just an update-bazel target?  I have to do that all the time

Reply:
I had a problem with loading the external dependencies when I first run the make targets. On fresh install on my machine until I did not add -- update-repos, it did not worked.